src/lib/__tests__/InstantSearch-status.test.ts

```diff
@@ -0,0 +1,213 @@
+import { createSearchClient } from '../../../test/mock/createSearchClient';
```

Comment:
Why make this a standalone file instead of a `describe` block in `InstantSearch-test.tsx`?

Reply:
the file is 2500 lines long, I thought it was too large and should be split up, this being a clear "unit" that fits in its own file.

Reply:
That makes perfect sense, but in this case I'd rename it `status.test.ts`. We don't really test the class, we test status scenarios.